services/eventhistory/README.md

```diff
@@ -0,0 +1,21 @@
+# Eventhistory service
+
+The `eventhistory` consumes all events from the configured event systems, stores them and allows to retrieve them via an eventid
+
+## Consuming
+
+The `eventhistory` services consumes all events from the configured event sytem. Running it without an event sytem is not possible.
+
+## Storing
+
+The `eventhistory` stores each consumed event in the configured store. Possible stores are:
+- `mem`: basic inmemory store and the default
+- `ocmem`: advanced inmemory store allowing max size
+- `redis`: stores data in a configured redis cluster
+- `etcd`: stores data in a configured etcd cluster
+- `nats-js`: stores data using nats
+- `noop`: stores nothing. Useful for testing. Not recommended in productive enviroments
+
+## Retrieving
+
+Other services can call the `eventhistory` service via a grpc call to retrieve events. The request must contain the eventid that should be retrieved
```

Comment:
```suggestion
# Eventhistory Service

The `eventhistory` consumes all events from the configured event system like NATS, stores them and allows to retrieve them via an eventid.

## Prerequisites

Running the eventhistory service without an event sytem like NATS is not possible.

## Consuming

The `eventhistory` services consumes all events from the configured event sytem.

## Storing

The `eventhistory` stores each consumed event in the configured store. Possible stores are:
- `mem`: basic inmemory store and the default
- `ocmem`: advanced inmemory store allowing max size
- `redis`: stores data in a configured redis cluster
- `etcd`: stores data in a configured etcd cluster
- `nats-js`: stores data using nats
- `noop`: stores nothing. Useful for testing. Not recommended in productive enviroments

## Retrieving

Other services can call the `eventhistory` service via a grpc call to retrieve events. The request must contain the eventid that should be retrieved.
```
The text should include somehow what clears history data. Is it time or retrieval or...?
`Other services can call...` is it only about services or could that include clients?

Reply:
Yes true. There is an envvar `EVENTHISTORY_RECORD_EXPIRY` which defines how long events survive in the store. But it should be mentioned here.